flow/project.py

```diff
@@ -282,10 +298,10 @@ class JobOperation(object):
         :class:`dict`
     """
 
-    def __init__(self, id, name, job, cmd, directives=None):
+    def __init__(self, id, name, jobs, cmd, directives=None):
         self._id = id
         self.name = name
-        self.job = job
+        self.jobs = jobs
```

Comment:
Make this private for now.

Reply:
Also, since we internally use this class check if the `job` parameter is a list, so we can store it appropriately. I know this is a bit more of a hassle for now.

Reply:
Will insert an assert statement